test/e2e_node/cpu_manager_test.go

```diff
@@ -144,8 +148,16 @@ func enableCPUManagerInKubelet(f *framework.Framework) (oldCfg *kubeletconfig.Ku
 	framework.ExpectNoError(err)
 	newCfg := oldCfg.DeepCopy()
 
-	// Enable CPU Manager using feature gate.
-	newCfg.FeatureGates[string(features.CPUManager)] = true
```

Comment:
This is not required anymore as the CPU manager feature is ON by default after graduation to Beta.